app/components/CapacityBars.tsx

```diff
@@ -23,7 +23,7 @@ export const CapacityBars = ({
   allocatedLabel: string
 }) => {
   return (
-    <div className="mb-12 flex min-w-min flex-col gap-3 lg+:flex-row">
+    <div className="1000:flex-row mb-12 flex min-w-min flex-col gap-3">
```

Comment:
These breakpoints I've added to the design system, previously we had a different style for this syntax each time. The width syntax here I find much easier since I don't have to remember how wide `sm` is.